react/BaseContent.tsx

```diff
@@ -36,33 +31,33 @@ const Content: FC<Props> = ({ finishShoppingButtonLink }) => {
     variation === 'drawer' ? 'pa4' : 'pv3'
   } sticky`
 
-  const emptyStateClasses = `${handles.minicartEmptyStateContainer} ${styles.minicartEmptyStateContainerDefault} pa9 flex justify-center`
+  const isEmptyCart = !loading && orderForm.items.length === 0
```

Comment:
Maybe `isCartEmpty` is better? I am not a 100% sure though

Reply:
Semantics, but are you checking for a empty cart or a cart that is empty 🤔 🤔 🤔 🤔 🤔

Reply:
Uuuhm... makes sense hahahah
I'm checking for a cart that is empty... I guess 😂

Reply:
🤔 💭 🤷‍♀